src/Laravel/config/api-platform.php

```diff
@@ -88,22 +99,22 @@
         //    'refreshUrl' => '',
         //    'scopes' => ['scope1' => 'Description scope 1'],
         //    'pkce' => true
-        //],
-        //'license' => [
+        // ],
+        // 'license' => [
         //    'name' => 'Apache 2.0',
         //    'url' => 'https://www.apache.org/licenses/LICENSE-2.0.html',
-        //],
-        //'contact' => [
+        // ],
+        // 'contact' => [
         //    'name' => 'API Support',
         //    'url' => 'https://www.example.com/support',
         //    'email' => 'support@example.com',
-        //],
+        // ],
     ],
 
     'url_generation_strategy' => UrlGeneratorInterface::ABS_PATH,
 
     'serializer' => [
         'hydra_prefix' => false,
         // 'datetime_format' => \DateTimeInterface::RFC3339
-    ]
+    ],
 ];
```

Comment:
I see this file modified but it looks like cs fixes that I should probably do in another PR. Would you be able to add the configuration at https://github.com/api-platform/core/blob/main/src/Laravel/ApiPlatformProvider.php#L1295 as well?

Reply:
Yes, cs fixer did it :)